### ISSUE TITLE
Formatter isn't modifying files or documentation is unclear.

### ISSUE THREAD

**Issue by pardom** [shyiko/ktlint#8]
The formatter is showing no errors, although errors exist.

Executing:

```shell
❯ ktlint -F --debug > ~/Desktop/format.txt
[DEBUG] Checking /Workspace/pardom/redux-kotlin/lib/src/main/kotlin/redux/Dispatcher.kt
[DEBUG] Checking /Workspace/pardom/redux-kotlin/lib/src/test/kotlin/redux/MiddlewareTest.kt
[DEBUG] Checking /Workspace/pardom/redux-kotlin/lib/src/test/kotlin/redux/StoreTest.kt
[DEBUG] Checking /Workspace/pardom/redux-kotlin/lib/src/main/kotlin/redux/Store.kt
[DEBUG] Checking /Workspace/pardom/redux-kotlin/lib/src/main/kotlin/redux/Middleware.kt
[DEBUG] Checking /Workspace/pardom/redux-kotlin/lib/src/main/kotlin/redux/Reducer.kt
[DEBUG] Checking /Workspace/pardom/redux-kotlin/lib/src/test/kotlin/redux/helpers/Reducers.kt
[DEBUG] Checking /Workspace/pardom/redux-kotlin/lib/src/test/kotlin/redux/helpers/ActionCreators.kt
[DEBUG] Checking /Workspace/pardom/redux-kotlin/lib/src/test/kotlin/redux/helpers/Todos.kt
[DEBUG] 766ms / 9file(s) / 0error(s)
```

and then:

```shell
❯ ktlint --debug > ~/Desktop/lint.txt
[DEBUG] Checking /Workspace/pardom/redux-kotlin/lib/src/test/kotlin/redux/helpers/ActionCreators.kt
[DEBUG] Checking /Workspace/pardom/redux-kotlin/lib/src/test/kotlin/redux/MiddlewareTest.kt
[DEBUG] Checking /Workspace/pardom/redux-kotlin/lib/src/test/kotlin/redux/StoreTest.kt
[DEBUG] Checking /Workspace/pardom/redux-kotlin/lib/src/main/kotlin/redux/Store.kt
[DEBUG] Checking /Workspace/pardom/redux-kotlin/lib/src/test/kotlin/redux/helpers/Todos.kt
[DEBUG] Checking /Workspace/pardom/redux-kotlin/lib/src/main/kotlin/redux/Dispatcher.kt
[DEBUG] Checking /Workspace/pardom/redux-kotlin/lib/src/main/kotlin/redux/Middleware.kt
[DEBUG] Checking /Workspace/pardom/redux-kotlin/lib/src/test/kotlin/redux/helpers/Reducers.kt
[DEBUG] Checking /Workspace/pardom/redux-kotlin/lib/src/main/kotlin/redux/Reducer.kt
[DEBUG] 625ms / 9file(s) / 222error(s)
```

[lint.txt](https://github.com/shyiko/ktlint/files/430626/lint.txt)

**Comment by shyiko** [comment]
Hi Michael,

Yeah, [DEBUG] 766ms / 9file(s) / **0error(s)** is obviously not correct. It will be fixed (along with format's standard output). Thank you.

Regarding errors: I cloned https://github.com/pardom/redux-kotlin and ran `ktlint`. ktlint reported 22**3** error (222 of which was "Unexpected indentation" and 1 "Unused import"). `ktlint -F` fixed "Unused import" but not "Unexpected indentation"s (`ktlint`'s formatter doesn't fix 2 types of issues at this point: wildcard imports & indentation; both can be taken care of by [Intellij *](https://github.com/shyiko/ktlint#-with-intellij-idea) formatter).

**Comment by pardom** [comment]
Okay, thanks.

**Comment by shyiko** [comment]
Fixed in 0.2.0.
